app/src/js/utils/table-config/collections.js

```diff
@@ -73,9 +60,10 @@ export const tableColumns = [
   {
     Header: 'MMT',
     accessor: 'MMTLink',
-    Cell: ({ cell: { value } }) => (value ? <a href={value} target="_blank">MMT</a> : null), // eslint-disable-line react/prop-types
+    Cell: ({ cell: { value } }) => (value ? <a href={value} target="_blank" rel="noopener noreferrer">MMT</a> : null), // eslint-disable-line react/prop-types
```

Comment:
What does this do? Curious about the behavior changes here.

Reply:
`rel="noopener noreferrer"` is a security best practice for `target="_blank"` and just noticed it when I was updating the links for the table.